frontend/src/screens/LoginScreen.js

```diff
@@ -32,51 +36,63 @@ const LoginScreen = ({ location, history }) => {
   };
 
   return (
-    <FormContainer>
-      <h1>Sign In</h1>
-      {error && <Message variant="danger">{error}</Message>}
-      {loading && <Loader />}
-      <Form onSubmit={submitHandler}>
-        <Form.Group controlId="email">
-          <Form.Label>Email Address</Form.Label>
-          <Form.Control
-            type="email"
-            placeholder="Enter email"
-            value={email}
-            onChange={(e) => setEmail(e.target.value)}
-          />
-        </Form.Group>
+    <StyledContainerDiv>
+      <StyledContainerDiv2>
+        <StyledHeader>LOGIN</StyledHeader>
+        {error && <Message variant="danger">{error}</Message>}
+        {loading && <Loader />}
+        <Form onSubmit={submitHandler}>
+          <Form.Group controlId="email">
+            <Form.Label>Email Address</Form.Label>
+            <Form.Control
+              type="email"
+              placeholder="Enter Email"
+              value={email}
+              onChange={(e) => setEmail(e.target.value)}
+            />
+          </Form.Group>
 
-        <Form.Group controlId="password">
-          <Form.Label>Password</Form.Label>
-          <Form.Control
-            type="password"
-            placeholder="Enter password"
-            value={password}
-            onChange={(e) => setPassword(e.target.value)}
-          />
-        </Form.Group>
+          <Form.Group controlId="password">
+            <Form.Label>Password</Form.Label>
+            <Form.Control
+              type="password"
+              placeholder="Enter Password"
+              value={password}
+              onChange={(e) => setPassword(e.target.value)}
+            />
+          </Form.Group>
 
-        <Button type="submit" variant="primary">
-          Sign In
-        </Button>
-      </Form>
+          <Button type="submit" variant="danger" block>
+            LOGIN
+          </Button>
+        </Form>
 
-      <Row className="py-3">
-        <Col>
-          New Customer?{' '}
-          <Link
-            to={
-              redirect
-                ? `/register?redirect=${redirect}`
-                : '/register'
-            }
-          >
-            Register
-          </Link>
-        </Col>
-      </Row>
-    </FormContainer>
+        <div style={{ textAlign: 'center' }}>
+          <Row className="py-3">
+            <Col>
+              <a href="/" style={{ color: 'black' }}>
+                <u>Forgot Password?</u>
```

Comment:
Don't use the \<u\> Tag for underline. It is not meant for that
[MDN \<u\> element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/u)